src/cmdlinerunner.zig

```diff
@@ -0,0 +1,229 @@
+const std = @import("std");
+const zware = @import("zware");
+
+fn oom(e: error{OutOfMemory}) noreturn {
+    @panic(@errorName(e));
+}
+
+const ImportStub = struct {
+    module: []const u8,
+    name: []const u8,
+    type: zware.FuncType,
+};
+
+const enable_leak_detection = false;
```

Comment:
Is this here because there is some issue with the memory allocation strategy in `zware` itself, or does this arise / is something illustrative for particular `.wasm` file?

Reply:
I think it's because there's one or more memory leaks in ZWare.